mne/channels/tests/test_montage.py

```diff
@@ -33,16 +33,13 @@
 from mne.io._digitization import (_format_dig_points,
                                   _get_fid_coords, _get_dig_eeg,
                                   _count_points_by_type)
-
 from mne.viz._3d import _fiducial_coords
-
 from mne.io.kit import read_mrk
 from mne.io import (read_raw_brainvision, read_raw_egi, read_raw_fif,
-                    read_fiducials, __file__ as _MNE_IO_FILE)
-
+                    read_fiducials, __file__ as _io_file)
 from mne.io import RawArray
 from mne.datasets import testing
-from mne.io.brainvision import __file__ as _BRAINVISON_FILE
+from mne.io.brainvision import __file__ as _io_brainvision_file
```

Comment:
I think we ignore these. I would only change things for the things we don't ignore.